Automattic/vip-go-mu-plugins
# Allow VIP File System uploads as email attachments

## Description
Extends PHPMailer to allow replacing of the `isPermittedPath()` to accommodate VIP File System

## Changelog Description

tbd

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

```
wp> $media_id = get_posts( [ 'post_type' => 'attachment', 'posts_per_page' => 1 ] )[0]->ID;
int(..)
wp> wp_mail( 'EMAIL_ADDRESS', 'Attachment test', 'Lorem Ipsum', '', [ get_attached_file( $media_id ) ] );
bool(true)
```

_Save yourself a testing headache and verify the media item actually exists._

**Before patch:** see that _Attachment test_ has arrived, without an attachment. 👎 
_Note: `wp_mail()` still sends, hence `true`_

**After patch:** see that _Attachment test_ has arrived, with an attachment. 👍
